fix(components/packages): only set `size` on icon component if `iconSize` not set

Blackbaud-SteveBrush opened the pull request.

Blackbaud-SteveBrush committed:
fix(components/packages): only set `size` on icon component if `iconSize` not set

blackbaud-sky-build-user commented:
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/3329/storybook/)

Component Storybooks:

- _(no component storybooks affected in this pr)_ 

Apps:

- [code-examples-playground](https://blackbaud.github.io/skyux-pr-preview/3329/code-examples-playground/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/3329/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/3329/playground/)

blackbaud-sky-build-user committed:
chore: changelog for 11.49.1 (#3328) 🍒 (#3330)

:cherries: Cherry picked from #3328 [chore: release
11.49.1](https://github.com/blackbaud/skyux/pull/3328)

Blackbaud-SteveBrush committed:
change test

Blackbaud-SteveBrush committed:
fix test

Blackbaud-SteveBrush committed:
Merge branch 'main' into fix-icon-schematic
